calculator.js

```diff
@@ -0,0 +1,31 @@
+//Add
+function [
+      function addition(num1, num2){
+            return num1 + num2;
+            console.log(addition(num1, num2));
+      }
+      function subtraction (num1, num2){
+            return num1 - num2;
+            console.log(subtraction(num1, num2));
+      }
+      function multiply (num1, num2){
+            return num1 * num2;
+            console.log(multiply(num1, num2));
+      }
+      function multiply (num1, num2){
+            return num1 * num2;
+            console.log(multiply(num1, num2));
+      }
+      function divide (num1, num2){
+            return num1 / num2;
+            console.log(divide(num1, num2));
+      }
+      function exponents Math.pow(){
+            return Math.pow
+            console.log(math.pow)
+      }
+]
+
```

Comment:
The instructions were trying to have you format your code into an object which would look like this:
```js
var calculator = {
  addition: function (num1, num2) {
    return num1 + num2
  },
  subtraction: function .....etc ....etc
}
```
Structuring like this allows you to call the functions like this:
```js
calculator.addition(2, 4)
// this will return 6
```